### DISCUSSION TITLE
SD img to img

### DISCUSSION THREAD

**Discussion by hisham-hchowdhu** [lshqqytiger/stable-diffusion-webui-directml#247] [General]
does the onnx or olive path work with img2img case of this webUI? do you have a recipe for img2img path work with onnx?

**Comment by Siralob** [comment]
Maybe, this is the short answer for now. 
https://github.com/lshqqytiger/stable-diffusion-webui-directml/issues/243#issuecomment-1689274529

**Reply by hisham-hchowdhu** [reply]
thanks @Siralob !! would be great to have it soon @lshqqytiger
